cves/kernel/CVE-2021-38166.yml

```diff
@@ -335,9 +350,10 @@ stacktrace:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  any_stacktraces:
-  stacktrace_with_fix:
-  note:
+  any_stacktraces: true
+  stacktrace_with_fix: true
+  note: |
+    The commit contains a stacktrace that leads to the method in concern.
```

Comment:
This sounds wrong to read for some reason, think the correct way to say it is either "method of concern" or "method in question" but maybe it's just me